common/firmware/motor_driver/motor.c

```diff
@@ -18,3 +19,7 @@ void reset_pin(struct PinConfig config) {
 }
 
 void delay(const int seconds) { vTaskDelay(seconds); }
+
+void start_it() { TIM7_Start_IT(); }
```

Comment:
Nit: Does `IT` stand for something (interrupt timer)? I read this as "start it". Can we call it by a full name?

Reply:
It stands for interrupt timer. We can change it to be a full name for non-HAL functions.